app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<% @recipes.each do |recipe| %>
+  <div class="row">
+    <div>
+      <h2> Recipe <%= recipe.id %> </h2>
+      <%= link_to "Delete", user_recipe_path(user_id: current_user.id, id: recipe.id), class: "btn btn-outline-danger", "data-turbo-method": :delete %>
+    </div>
+    <p> <%= recipe.description %> </p>
+  </div>
+<% end %>
+
+<%= link_to "Create", new_user_recipe_path(user_id: current_user.id), class: "btn btn-outline-success" %>
```

Comment:
- [ ] [optional] Create button would be more visible and useful if it is at the top of the page. In the instance of many recipes, the user would need to go to the bottom of the page.